src/reactive-handler.ts

```diff
@@ -37,21 +35,21 @@ function unwrapDescriptor(descriptor: PropertyDescriptor): PropertyDescriptor {
 }
 
 function lockShadowTarget(membrane: ReactiveMembrane, shadowTarget: ReactiveMembraneShadowTarget, originalTarget: any): void {
-    const targetKeys = ArrayConcat.call(getOwnPropertyNames(originalTarget), getOwnPropertySymbols(originalTarget));
+    const targetKeys = getOwnKeysAndMagicProperty(originalTarget, membrane.magicPropertyKey);
     targetKeys.forEach((key: PropertyKey) => {
-        let descriptor = getOwnPropertyDescriptor(originalTarget, key) as PropertyDescriptor;
```

Comment:
removing the casting... in favor of an if condition, this also help with the case were one key is not in the original target because it is the magic key